examples/ads.amp.html

```diff
@@ -247,6 +248,30 @@ <h2 id="adgeneration">Ad Generation</h2>
     <div fallback></div>
   </amp-ad>
 
+  <h2 id="adhese">Adhese</h2>
+  <amp-ad width=300 height=250
+      type="adhese"
+      data-location="_sdk_amp_"
+      data-position=""
```

Comment:
sorry meant to ask `data-position` 
So `data-position` is required according to the doc, but it can be empty? A bit confusing to me.